superset/assets/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -250,6 +250,7 @@ class FilterBox extends React.Component {
         selectComponent={Creatable}
         selectWrap={VirtualizedSelect}
         optionRenderer={VirtualizedRendererWrap(opt => opt.label)}
+        noResultsText={t('No results found')}
```

Comment:
I think it would be worth adding this as an optional prop to `OnPasteSelect` so that we get some type safety

https://github.com/apache/incubator-superset/blob/master/superset/assets/src/components/OnPasteSelect.jsx#L95

Reply:
Ok, fixed

Reply:
Ok, fixed